File: Assignments September 2024/Lp4-1.rb
puts "# of copies to print"
a = gets.chomp.to_i

if a > 99
    price = 0.28
elsif a > 499
    price = 0.27
elsif a > 749
    price = 0.26
elsif a > 1000
    price = 0.25
else 
    price = 0.30
end 

total = a * price
price = price.to_s
total = total.to_s
a = a.to_s
puts "The # of copies are: " + a
puts "The price per copy is: " + price
puts "The total cost will be is: " + total

File: Python-practice/Lp4_1.py

def pricing(x):
    if x > 99:
        y = 0.28 #Price
    elif x > 499:
        y = 0.27
    elif x >749:
        y = 0.26
    elif x > 1000:
        y = 0.25
    else:
        y = 0.30
    return y

def report(x, y):
    z = x * y
    print("Price for each copy: "+str(y)+"$.")
    print("Amount of copies requested: "+str(x)+"$.")
    print("Total cost for copies requested: "+str(z)+"$.")
    return z #so the total is avialable outside this function

a = int(input("How many copies do you want to print?"))
if a > 0:
    pricing(a) # type: ignore
else:
    print("what are you doing here then")

